lib/flipper/views/feature.erb

```diff
@@ -208,7 +208,7 @@
                           <%== csrf_input_tag %>
                           <input type="hidden" name="value" value="<%= item %>">
                           <button class="va-button-link vads-u-margin--0" type="submit" name="operation" value="disable" title="Disable <%= item %>">
-                            <span class="octicon octicon-x"></span>
+                            <span class="octicon octicon-x">remove</span>
```

Comment:
This might look a bit nicer.
```suggestion
                            <span class="octicon octicon-x">Remove</span>
```

Reply:
@rmtolmach I'm not aware of any edge cases, except disabling the toggle for everyone should remove all of the users added.  I updated the text to 'Remove'.